src/services/gnosis/constants.ts

```diff
@@ -0,0 +1,25 @@
+import { IS_DEV } from '@/constants/env';
+import { APP_NETWORK_ID, Network } from '@/constants/network';
+import {
+  GPv2Settlement,
+  GPv2AllowanceManager
+} from '@gnosis.pm/gp-v2-contracts/networks.json';
+
+export const OPERATOR_URLS = {
+  [Network.MAINNET]: IS_DEV
+    ? 'https://protocol-mainnet.dev.gnosisdev.com/api'
+    : 'https://protocol-mainnet.gnosis.io/api',
+  [Network.RINKEBY]: IS_DEV
+    ? 'https://protocol-rinkeby.dev.gnosisdev.com/api'
+    : 'https://protocol-rinkeby.gnosis.io/api'
+};
+
+export const OPERATOR_URL =
+  OPERATOR_URLS[APP_NETWORK_ID] ?? OPERATOR_URLS[Network.MAINNET];
+
+export const GP_SETTLEMENT_CONTRACT_ADDRESS = GPv2Settlement[APP_NETWORK_ID]
```

Comment:
for me, this fails on Kovan if I run it locally:
```
Uncaught TypeError: Cannot read property 'address' of undefined
    at Module.cd49 (constants.ts:21)
    at c (bootstrap:84)
    at Object.0 (app.b3566afb.js:1)
    at c (bootstrap:84)
    at n (bootstrap:45)
    at bootstrap:221
    at app.b3566afb.js:1
```

not sure if problem is on my side